platform_secrets/service.py

```diff
@@ -1,28 +1,97 @@
 from __future__ import annotations
 
+import hashlib
 import logging
+import math
 import re
 from dataclasses import dataclass, field
-from typing import Optional
+from typing import Any, Optional
 
-from .kube_client import (
-    KubeClient,
+from apolo_kube_client.errors import (
     ResourceBadRequest,
-    ResourceConflict,
+    ResourceExists,
     ResourceInvalid,
     ResourceNotFound,
 )
 
+from .kube_client import KubeApi
+
 logger = logging.getLogger()
 
 SECRET_API_ORG_LABEL = "platform.neuromation.io/secret-api-org-name"
 APPS_SECRET_NAME = "apps-secrets"
 
 NO_ORG = "NO_ORG"
+NO_ORG_NORMALIZED = "no-org"
 
 NAMESPACE_ORG_LABEL = "platform.apolo.us/org"
 NAMESPACE_PROJECT_LABEL = "platform.apolo.us/project"
 
+KUBE_NAME_LENGTH_MAX = 63
```

Comment:
afair kubernetes namespace max length is 253 characters. Or do we want to have max 63 characters?

Reply:
@zubenkoivan I think it was changed:

```
* metadata.name: Invalid value: "ed05cdec0d284b04a2c13ace985479f0ed05cdec0d284b04a2c13ace985479f0ed05cdec0d284b04a2c13ace985479f0": must be no more than 63 characters
```